commit 15b00d2af0e56dcc8c244a36d6872d301b0c7185
Author: Noah Goldstein <goldstein.w.n@gmail.com>
Date:   Fri Feb 18 17:00:25 2022 -0600

    x86: Fix TEST_NAME to make it a string in tst-strncmp-rtm.c
    
    Previously TEST_NAME was passing a function pointer. This didn't fail
    because of the -Wno-error flag (to allow for overflow sizes passed
    to strncmp/wcsncmp)
    
    Reviewed-by: H.J. Lu <hjl.tools@gmail.com>
    (cherry picked from commit b98d0bbf747f39770e0caba7e984ce9f8f900330)

diff --git a/sysdeps/x86/tst-strncmp-rtm.c b/sysdeps/x86/tst-strncmp-rtm.c
index 4e9f094f39c72f67..aef9866cf2fbe774 100644
--- a/sysdeps/x86/tst-strncmp-rtm.c
+++ b/sysdeps/x86/tst-strncmp-rtm.c
@@ -23,12 +23,12 @@
 # define CHAR wchar_t
 # define MEMSET wmemset
 # define STRNCMP wcsncmp
-# define TEST_NAME wcsncmp
+# define TEST_NAME "wcsncmp"
 #else /* !WIDE */
 # define CHAR char
 # define MEMSET memset
 # define STRNCMP strncmp
-# define TEST_NAME strncmp
+# define TEST_NAME "strncmp"
 #endif /* !WIDE */
 
 
